### ISSUE TITLE
Better CLI for Executioner

### ISSUE THREAD

**Issue by Daniel5055** [CP-STA/STAOJ#38]
The executioner is designed to accept many useful options which can change how the executioner operates. Currently, calling `npm start` will only use the  default values for many options like executing one submission at a time and setting the tmp submission directory to `tmp` inside the executioner directory.

Having a means for which the executioner can be called with different command line arguments to set different options would allow for users to customise their executioner experience to their needs. Additionally, different 'presets' could also be defined in the package run scripts to run the executioner with different sets of predefined arguments.

**Comment by DE0CH** [comment]
Is this really needed? Presets might be useful but for further customisation they can always just modify the source code.

**Comment by Daniel5055** [comment]
True, but at the very least, allowing for specifying options like how many submissions to execute simultaneously (e.g how many cpu cores someone has) is useful.
